src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -133,9 +156,27 @@ protected override void PostPersist(ApplicationEngine engine)
 
             int index = (int)(engine.Snapshot.PersistingBlock.Index % (uint)ProtocolSettings.Default.CommitteeMembersCount);
             var gasPerBlock = GetGasPerBlock(engine.Snapshot);
-            var pubkey = GetCommittee(engine.Snapshot)[index];
+            var committee = GetCommittee(engine.Snapshot);
+            var pubkey = committee[index];
             var account = Contract.CreateSignatureRedeemScript(pubkey).ToScriptHash();
             GAS.Mint(engine, account, gasPerBlock * CommitteeRewardRatio / 100);
+
+            // Record the cumulative reward of the voters of committee
+
+            var m = ProtocolSettings.Default.CommitteeMembersCount + ProtocolSettings.Default.ValidatorsCount;
+            var cindex = engine.Snapshot.PersistingBlock.Index % m % ProtocolSettings.Default.CommitteeMembersCount;
```

Comment:
@Tommo-L What about changing the epoch to 21 blocks?

```c#
if (cindex < ProtocolSettings.Default.ValidatorsCount)
{
    //double the reward
}
```

Reply:
Then we can unify the epoch of committee's reward and voters' reward.

Reply:
Sure. The only downside is that the `voter`  rewards of each block are not complete, which equals `gasPerBlock * 21/28 * 85%`  , not `gasPerBlock * 85%`.

I'll adjust it.

Reply:
> The only downside is that the voter rewards of each block are not complete, which equals gasPerBlock * 21/28 * 85% , not gasPerBlock * 85%.

Why?

Reply:
Done. 

Now, one epoch has 21 blocks. https://github.com/neo-project/neo/pull/1848/commits/94caa538e90333f5d3a66585ecd1c8f2562816e7